tutorials/plot_metadata_epochs.py

```diff
@@ -1,4 +1,6 @@
 """
+.. _tut_metadata_epochs:
```

Comment:
don't add this, there is a standard ref tag automagically inserted based on filename `sphx_glr_auto_tutorials_plot_metadata_epochs.py`

Reply:
Ugh, how did this end up there? Must have come with the rebase or something. I didn't do it manually.